boards/nxp/fmurt1170-v1/nuttx-config/scripts/script.ld

```diff
@@ -88,10 +88,11 @@ SECTIONS
         *(.got)
         *(.gcc_except_table)
         *(.gnu.linkonce.r.*)
-        _etext = ABSOLUTE(.);
         . = ALIGN(4096);
+        _etext = ABSOLUTE(.);
         _srodata = ABSOLUTE(.);
         *(.rodata .rodata.*)
+        . = ALIGN(4096);
```

Comment:
Do you need both?

Reply:
Start and end address need to be 4K aligned.
First ALIGN enforces that both _etext and _srodata are 4k aligned.
2nd ALIGN enforces that _erodata is 4k aligned.